syntax = "proto2";

package kythe.proto;
option java_package = "com.google.devtools.kythe.proto";

import "kythe/proto/storage.proto";

// An assignment of analysis responsibility.
message ClaimAssignment {
  // The compilation responsible for analyzing the dependency.
  optional VName compilation_v_name = 1;
  // The dependency that needs analyzed.
  optional VName dependency_v_name = 2;
}
